datalad_dataverse/create_sibling_dataverse.py

```diff
@@ -68,10 +68,30 @@ class CreateSiblingDataverse(Interface):
     Dataverse is a web application to share and cite research data.
     Research data published in Dataverse receives an academic citation which allows to grant full credit and increases visibility of your work.
     
+    In order to be able to use this command, a personal access token has to be generated on the Dataverse platform. You can find it by clicking on your name at the top right corner, and then clicking on Api Token>Create Token.
+
     TODO: command doc
     """
 
-    _examples_ = []
+    _examples_ = [
+        dict(text="Use a new sibling on GIN as a common data source that is "
+                  "auto-available when cloning from GitHub",
```

Comment:
I think the description here is not yet what you want it to be :) You can make it something generic like "Create a dataset sibling in the form of a dataverse dataset"

Reply:
oh true. thanks

Reply:
I changed and pushed